src/main/java/nl/avans/drivio/repository/UserRepository.java

```diff
@@ -1,59 +1,7 @@
 package nl.avans.drivio.repository;
 
 import nl.avans.drivio.model.User;
-import org.hibernate.Session;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Repository;
-import org.springframework.transaction.annotation.Transactional;
+import org.springframework.data.jpa.repository.JpaRepository;
 
-import javax.persistence.EntityManager;
-import java.util.List;
-
-@Repository
-public class UserRepository implements IUserRepository {
```

Comment:
Moet de naam van het bestand en de klasse net IUserRepository zijn?

Reply:
Dit gaan we fixen in issue #27